core/src/main/scala/com/raphtory/algorithms/api/GenericTable.scala

```diff
@@ -17,11 +17,15 @@ class GenericTable(val query: Query, private val querySender: QuerySender) exten
     addFunction(Explode(closurefunc))
   }
 
-  override def writeTo(outputFormat: OutputFormat, jobName: String = ""): QueryProgressTracker = {
+  override def writeTo(outputFormat: OutputFormat, jobName: String): QueryProgressTracker = {
     val query = addFunction(WriteTo(outputFormat)).query
     querySender.submit(query, jobName)
   }
 
+   override def writeTo(outputFormat: OutputFormat): QueryProgressTracker = {
```

Comment:
Ahhh is this because python can't deal with the default arguments?

Reply:
yes, py4j uses java bindings of the scala code which do not have default args